File: src/demos/ps/maps/Aukba_entrance.java
package demos.ps.maps;

import static demos.ps.oo.PSGame.mapswitch;
import demos.ps.oo.City;
import demos.ps.oo.Dungeon;
import demos.ps.oo.PSGame.Planet;

public class Aukba_entrance {


	public static void tunnel() {
		mapswitch(Dungeon.AUKBA_TUNNEL_IN);
	}
	
	public static void exit() {
		mapswitch(Planet.DEZORIS, City.AUKBA_ENTRANCE.getX(),City.AUKBA_ENTRANCE.getY());
	}

}

File: src/demos/ps/maps/Aukba_entrance.ts
/**
 * Aukba_entrance Script
 * TypeScript port of Aukba_entrance.java
 */

import { PSGame } from '../PSGame';
import { Planet, City, CityHelper } from '../game/City';
import { Dungeon } from '../game/Dungeon';

export class Aukba_entrance {

  public static async tunnel(): Promise<void> {
    await PSGame.mapswitchToDungeon(Dungeon.AUKBA_TUNNEL_IN);
  }

  public static exit(): void {
    PSGame.mapswitchToPlanet(Planet.DEZORIS, CityHelper.getX(City.AUKBA_ENTRANCE), CityHelper.getY(City.AUKBA_ENTRANCE));
  }
}
